packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/metadata.py

```diff
@@ -45,28 +45,38 @@ class ParameterMetadata(NamedTuple):
     field_name: str = ""
     """display_name in snake_case format."""
 
-
-def initialize(
-    display_name: str,
-    type: type_pb2.Field.Kind.ValueType,
-    repeated: bool,
-    default_value: Any,
-    annotations: Dict[str, str],
-    message_type: str = "",
-) -> ParameterMetadata:
-    """Initialize ParameterMetadata with field_name."""
-    underscore_display_name = display_name.replace(" ", "_")
-    if all(char.isalnum() or char == "_" for char in underscore_display_name):
-        field_name = underscore_display_name
-    else:
-        field_name = "".join(
-            char for char in underscore_display_name if char.isalnum() or char == "_"
+    enum_type: Any = None
```

Comment:
The correct type hint is here: https://github.com/ni/measurement-plugin-python/blob/a6bc38ea4fbe7f6dc624e752382b681aecbda6f6/packages/service/ni_measurement_plugin_sdk_service/measurement/service.py#L71

Here's what to put in metadata.py:

```python
from __future__ import annotations

...

if TYPE_CHECKING:
    from google.protobuf.internal.enum_type_wrapper import _EnumTypeWrapper

    SupportedEnumType = Union[Type[Enum], _EnumTypeWrapper]

...

enum_type: Optional[SupportedEnumType] = None
```

from __future__ import annotations allows type hints to use types like SupportedEnumType that only exist at type check time, not at run time.